fraim/scan.py

```diff
@@ -41,23 +41,17 @@ def resolve_file_patterns(args: ScanArgs) -> List[str]:
         return WorkflowRegistry.get_file_patterns_for_workflows(args.workflows)
 
 
-def get_files(args: ScanArgs, config: Config) -> Tuple[str, Files]:
+def get_files_context(args: ScanArgs, config: Config) -> Files:
     """Get the local root path of the project and the files to scan."""
     file_patterns = resolve_file_patterns(args)
     config.logger.info(f"Using file patterns: {file_patterns}")
     if args.limit is not None:
         config.logger.info(f"File limit set to {args.limit}")  # TODO: enforce this
     if args.repo:
-        temp_dir = tempfile.mkdtemp(prefix="fraim_scan_")
-        repo_path = os.path.join(temp_dir, "repo")
```

Comment:
@prestonprice57 why are we creating a "repo" directory inside the temporary directory, can we not just use the temporary directory? Or are we also storing reports etc in the same temporary directory, i.e. `project_path`? 

As a user I would expect any file output to be written to the working directory.